tests/integration/console_scripts_test.py

```diff
@@ -95,15 +78,13 @@ def test_extract_example_filings(script_runner, tmp_path, test_dir):
     assert ret.success
 
     # Sanity check the sqlite/duckdb outputs
-    sqlite_uri = f"sqlite:///{sqlite_path.absolute()}"
-    sqlite_engine = create_engine(sqlite_uri)
     with (
-        sqlite_engine.begin() as sqlite_conn,
+        duckdb.connect(sqlite_path) as sqlite_conn,
```

Comment:
`duckdb` is happy to read SQLite, so we can just use `duckdb` for everything. This is what allowed me to delete the `_get_sqlite_tables` method.